src/types.jl

```diff
@@ -845,6 +845,8 @@ function show_term(io::IO, t)
         show_pow(io, args)
     elseif f === (getindex)
         show_ref(io, f, args)
+    elseif f === (identity)
```

Comment:
So @shashi and @YingboMa, you're suggesting this?

```suggestion
    elseif f === (identity) && !issym(args[1])
```

If so, I have no problem with this.

Reply:
Also !istree(args[1]]

Reply:
There's a syntax error. Could you also add a little test?

Reply:
Will do.